packages/apps/composer-app/vite.config.ts

```diff
@@ -18,6 +18,9 @@ import { ConfigPlugin } from '@dxos/config/vite-plugin';
 import { ThemePlugin } from '@dxos/react-ui-theme/plugin';
 
 import { appKey } from './src/constants';
+import IconsPlugin from "@ch-ui/vite-plugin-icons";
+
+const phosphorIconsCore = resolve(__dirname, '../../../node_modules/@phosphor-icons/core/assets')
```

Comment:
curious why this isn't in composer's deps but instead the monorepo's

Reply:
I wasn’t sure about the best place for this… certainly amenable to moving it if needed. Other packages are likely to need this, so this seemed like the most suitable place for it.